test/MediatR.Tests/MicrosoftExtensionsDI/Handlers.cs

```diff
@@ -176,7 +176,11 @@ class InternalPingHandler : IRequestHandler<InternalPing>
     }
 
     class MyCustomMediator : IMediator
-    {
+    {
+        public MyCustomMediator() { }
+        public MyCustomMediator(IServiceProvider provider) { }
+        public MyCustomMediator(IServiceProvider provider, INotificationPublisher publisher) { }
```

Comment:
You have added parameters (`provider` and `publisher`) in the constructor, but those parameters don't have any usage! Why is it needed?